python/src/la.cpp

```diff
@@ -674,6 +674,10 @@ void la(py::module& m)
       .def("update_ghost_values", &dolfin::la::PETScVector::update_ghost_values)
       .def("size", (std::size_t(dolfin::la::PETScVector::*)() const)
                        & dolfin::la::PETScVector::size)
+      .def("sum", &dolfin::la::PETScVector::sum)
```

Comment:
Do we want these wrappers, or just use petsc4py? @chrisrichardson ?

Reply:
I'm happy for these to be removed, depending on @chrisrichardson's opinion also.